include/SFML/Graphics/Transform.inl

```diff
@@ -64,25 +63,20 @@ constexpr Transform Transform::getInverse() const
 {
     // clang-format off
     // Compute the determinant
-    const float det = m_matrix[0] * (m_matrix[15] * m_matrix[5] - m_matrix[7] * m_matrix[13]) -
-                      m_matrix[1] * (m_matrix[15] * m_matrix[4] - m_matrix[7] * m_matrix[12]) +
-                      m_matrix[3] * (m_matrix[13] * m_matrix[4] - m_matrix[5] * m_matrix[12]);
+    const float det = m_matrix[0] * m_matrix[5] - m_matrix[1] * m_matrix[4];
```

Comment:
Also, have you considered just... checking here and branching?

```cpp
if (matrix[3] == 0.f && matrix[7] == 0.f && matrix[15] == 1.f)
{
    // special optimized case
}
else
{
    // original code here
}
```

This might be a good optimization that doesn't affect `sf::Transform`'s flexibility, but you'd have to prove it's worth it via benchmark.

Reply:
Adding a branch on the hot path does not seem like good idea.